packages/oshu/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/fmang/oshu
+TERMUX_PKG_DESCRIPTION="Oshu Is a Musical Game"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@AnGelXoG"
+TERMUX_PKG_VERSION=2.0.2
+TERMUX_PKG_SRCURL=https://github.com/fmang/oshu/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=2f576ffd87af58bc246f7e3277ec4193799648b56909a309990ba73af6070ad4
+TERMUX_PKG_DEPENDS="doxygen, ffmpeg, zstd, xorgproto, libcairo, pango, sdl2, sdl2-image"
```

Comment:
Specifying `doxygen` is unnecessary here. `xorgproto` is better in `TERMUX_PKG_BUILD_DEPENDS`.

Reply:
A ok.. :D